### DISCUSSION TITLE
Castbar target name?

### DISCUSSION THREAD

**Discussion by esurm** [siweia/NDui#139] [Q&A] [answered]
Hi!

I really love the aesthetic of NDui castbar, but I'm torn between it and something like Tidy Plates, because the latter offers cast target name option on castbar, which is huge for pvp and sometimes pve too.

Any chance to add it? Not so familiar with lua, but perhaps it's not a huge task?

PS There's a typo with latest update, it says "Marjor Spells Glow" instead of Major on the nameplate options 📦 
Cheers!

**Comment by siweia** [comment, marked as answer]
Already has.
